tools/ci_build/github/apple/default_full_macos_ios_framework_build_settings.json

```diff
@@ -0,0 +1,28 @@
+{
+    "build_osx_archs": {
+        "macosx": [
+            "x86_64"
+        ]
+    },
+    "build_params_for_macos": [
+        "--parallel",
+        "--use_xcode",
+        "--build_apple_framework",
+        "--use_coreml",
+        "--use_xnnpack",
+        "--skip_tests",
+        "--cmake_extra_defines=onnxruntime_BUILD_UNIT_TESTS=OFF",
+        "--apple_deploy_target=10.14"
```

Comment:
not sure what's the current/correct minimum support version for macos right now. put "10.14" for now

Reply:
it may be 11.0. but we can start with 10.14.

https://github.com/microsoft/onnxruntime/pull/18251

Reply:
that PR got merged so I guess the minimum MacOS version is 11.0 now.